althea_kernel_interface/src/open_tunnel.rs

```diff
@@ -6,7 +6,7 @@ use std::path::Path;
 use failure::Error;
 
 #[derive(Debug, Fail)]
-pub enum TunnelOpeningError {
+pub enum _TunnelOpeningError {
```

Comment:
think we should just delete this if we are not using it

Reply:
+1